src/scripts/scripts/zone/caverns_of_time/old_hillsbrad/boss_captain_skarloc.cpp

```diff
@@ -287,7 +287,7 @@ struct boss_captain_skarlocAI : public ScriptedAI
         else
             DevotionAura_Timer -= diff;
 
-        if (HeroicMode)
+        //Consecration (Heroic)
```

Comment:
How do we want to process? 

Changing `if (Consecration_Timer < diff)` to `if (Consecration_Timer && Consecration_Timer <= diff)` or readding heroic check? Readding heroic check seems to be more simple, clean and understandable to me at the moment.